manifests/t/ThaUnknown/Miru/3.1.7/ThaUnknown.Miru.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: ThaUnknown.Miru
+PackageVersion: 3.1.7
+PackageLocale: en-US
+Publisher: ThaUnkown

```

```suggestion
Publisher: ThaUnknown_
```
This must match Control Panel.

---

Ok, I wasn't sure if I had to remove any special characters. But then we indeed can keep it.